### ISSUE TITLE
RO-Crate extensions

### ISSUE THREAD

**Issue by HughCraig** [HughCraig/GHAP#172]
I am marvelling at the RO-Crate button and the super-convenient package that it creates.
I am wondering about the extension to this and how we should discuss it in documentation on the site.
As I understand it, RO-Crate can also support contextual information about the data files, e.g. a description of the fields included. If the heading for one of the columns in a spreadsheet is "Conditions", an info file would explain what this means and relate it to some standard terms in a schema. The Describo app does this, I believe, i.e. allows the user to supply this sort of information and wraps it up in a conformant way.
Do we simply tell users they can take the RO-Crates created in GHAP, import them into Describo, and add more explanation there? (Assuming that is something you can do.) What GHAP currently does is the maximum that can be done without more user input, but is not the ultimate stage of an RO-Crate, which would be a truly self-sufficient package, so the mythical Martian coming across the RO-Crate could understand what the data columns mean without already knowing anything about them.
I assume that the RO-Crates which GHAP can already create are conformant to the RO-Crate standard, so we are not required to do anything else, but think that the avenue to a more fully documented RO-Crate should be articulated.

**Comment by IanMcCrabb** [comment]
That seemed to be the sensible approach at the moment.  That a researcher might use Describo (or whatever comes out of the developments by ATAP et al in this area) to augment the descriptions rather than invest in UI surfaces for this in GHAP.

**Comment by HughCraig** [comment]
Yes, we can do our part with an automated RO-Crate and ideally users who wanted to add to that could use some other system like Describo developed for the purpose. I'm tempted to contact Peter Sefton or Marco La Rosa about this, but we may have a bundle of questions for them, including #173, and it could be better to do them all at one time rather than in dribs and drabs.

**Comment by BillPascoe** [comment]
Yes. When we release this we should have new doco, and a piece in the newsletter explaining:
- Now you can download layers, and multilayers as ROCrates. This is recommended for depositing a standards compliant archive of information in data repository for long term storage. TLCMap is not a repository but an application for creating and sharing humanities maps, we encourage you to back up and archive your data using this format. An ROCrate is a compressed package containing the data in open standard formats and a metadata description of it.
- You can open and edit the ROCrate to add or change information in it, or create your own ROCrates using in Describo.

**Comment by BillPascoe** [comment]
I have a job assigned to me for this. I need to add some doco to the guides online.
